modular_skyrat/modules/command_vendor/code/vending/vending.dm

```diff
@@ -10,61 +10,123 @@
 
 /obj/machinery/vending/access/command/build_access_list(list/access_lists)
 	access_lists["[ACCESS_CAPTAIN]"] = list(
-		/obj/item/clothing/head/caphat = 1, /obj/item/clothing/head/caphat/beret = 1, /obj/item/clothing/head/caphat/beret/alt = 1,
-		/obj/item/clothing/under/rank/captain = 1, /obj/item/clothing/under/rank/captain/skirt = 1, , /obj/item/clothing/under/rank/captain/humble = 1,
-		/obj/item/clothing/under/rank/captain/dress = 1, /obj/item/clothing/head/caphat/parade = 1,
-		/obj/item/clothing/under/rank/captain/parade = 1, /obj/item/clothing/suit/toggle/captains_parade = 1,
-		/obj/item/clothing/suit/captunic = 1, /obj/item/storage/backpack/captain = 1, /obj/item/storage/backpack/satchel/cap = 1,
-		/obj/item/storage/backpack/duffelbag/captain = 1, /obj/item/clothing/shoes/sneakers/brown = 1
+		// CAPTAIN
+		/obj/item/clothing/head/caphat = 1,
+		/obj/item/clothing/head/caphat/beret = 1,
+		/obj/item/clothing/head/caphat/beret/alt = 1,
+		/obj/item/clothing/under/rank/captain = 1,
+		/obj/item/clothing/under/rank/captain/skirt = 1,
+		/obj/item/clothing/under/rank/captain/humble = 1,
+		/obj/item/clothing/under/rank/captain/dress = 1,
+		/obj/item/clothing/head/caphat/parade = 1,
+		/obj/item/clothing/under/rank/captain/parade = 1,
+		/obj/item/clothing/suit/toggle/captains_parade = 1,
+		/obj/item/clothing/suit/captunic = 1,
+		/obj/item/clothing/neck/mantle/capmantle = 1,
+		/obj/item/storage/backpack/captain = 1,
+		/obj/item/storage/backpack/satchel/cap = 1,
+		/obj/item/storage/backpack/duffelbag/captain = 1,
+		/obj/item/clothing/shoes/sneakers/brown = 1,
+
+		// BLUESHIELD
```

Comment:
Does the Blueshield not have a unique access?

Reply:
yea, this really needs to be fixed. If its read like this, this means captain can access blueshield stuff.

Reply:
> yea, this really needs to be fixed. If its read like this, this means captain can access blueshield stuff.

I mean, to be fair, it's not inherently a bad thing.